env-configuration/requirements.txt

```diff
@@ -5,4 +5,7 @@ sdv
 uuid
 dython
 DataSynthesizer
-featuretools
\ No newline at end of file
+featuretools
+tensorboard==2.0.1
+tensorflow==2.0.0a0
+tensorflow-estimator==2.0.1
```

Comment:
We need to update the docker container with all these at some point - not immendiate though

Reply:
👍